src/stream_base-inl.h

```diff
@@ -6,7 +6,6 @@
 #include "stream_base.h"
 
 #include "node.h"
-#include "env-inl.h"
```

Comment:
So … this one is okay, but it works only because the includes for `stream_base.h` are already broken – it includes `async_wrap-inl.h`, but regular header files should not include `-inl.h` header files.

I think the way to fix that would be to move the `async_wrap-inl.h` include here, and instead only include `env.h` and `async_wrap.h` from the main `stream_base.h` header. Feel free to give that a try, if you like.

Reply:
Yep, I actually had that changed noted but hadn't gotten to it yet. The reason this is here is exactly because of the stream_base.h issue.

Reply:
Done.